src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Test/ScenarioTests/TestsBase.cs

```diff
@@ -12,38 +12,57 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models;
+using Microsoft.Azure.Management.RecoveryServices.Backup;
 using Microsoft.Azure.Test;
+using Microsoft.Azure.Test.Authentication;
 using Microsoft.Azure.Test.HttpRecorder;
 using Microsoft.WindowsAzure.Commands.ScenarioTest;
 using System;
-using System.Configuration;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using System.Net;
 using System.Net.Http;
 using System.Net.Security;
 using System.Reflection;
-using Microsoft.WindowsAzure.Commands.Test.Utilities.Common;
-using Microsoft.Azure.Management.RecoveryServices.Backup;
-using Hyak.Common;
-using Microsoft.Azure.Commands.Common.Authentication;
-using System.Collections.Generic;
-using System.IO;
-using Microsoft.Azure.Test.Authentication;
+using HyakRmNS = Microsoft.Azure.Management.Internal.Resources;
+using RecoveryServicesNS = Microsoft.Azure.Management.RecoveryServices;
+using ResourceManagementNS = Microsoft.Azure.Management.Resources;
+using ResourceManagementRestNS = Microsoft.Azure.Management.ResourceManager;
+using RestTestFramework = Microsoft.Rest.ClientRuntime.Azure.TestFramework;
 
 namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Test.ScenarioTests
 {
-    public abstract class TestsBase : RMTestBase
+    public class TestController
     {
         CSMTestEnvironmentFactory csmTestFactory;
         EnvironmentSetupHelper helper;
 
-        public RecoveryServicesBackupManagementClient RsBackupClient { get; private set; }
+        public RecoveryServicesBackupClient RsBackupClient { get; private set; }
+
+        public RecoveryServicesNS.RecoveryServicesManagementClient RsClient { get; private set; }
+
+        public ResourceManagementNS.ResourceManagementClient RmClient { get; private set; }
+
+        public ResourceManagementRestNS.ResourceManagementClient RmRestClient { get; private set; }
+
+        public HyakRmNS.ResourceManagementClient HyakRmClient { get; private set; }
```

Comment:
Check if we need to use same hydra dll references or swagger dll references for other SDKs like Storage/RG. Open a bug for tracking if we are not doing it now